java-agent/instrumentation/reactor-core-3.1/src/main/java8/io/opentelemetry/auto/instrumentation/reactor/core/ReactorCoreDecorator.java

```diff
@@ -0,0 +1,26 @@
+package io.opentelemetry.auto.instrumentation.reactor.core;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.auto.api.SpanTypes;
+import io.opentelemetry.auto.decorator.BaseDecorator;
+import io.opentelemetry.trace.Tracer;
+
+public class ReactorCoreDecorator extends BaseDecorator {
```

Comment:
Not needed with above change

Reply:
The ```ReactorCoreDecorator```ended up being needed anyway, since other parts of the code were referring to the Tracer and putting it in the decorator avoids muzzling problems.